### ISSUE TITLE
removeEnumPrefix doesn't work with enum defined in messages

### ISSUE THREAD

**Issue by Frabat** [stephenh/ts-proto#789]
Hi, I was trying to generate some enums and I've noticed that when an enum is defined in a message, removeEnumPrefix doesn't work.

```
enum Foo {
    FOO_UNSPECIFIED = 0;
    FOO_BAR = 1;
    FOO_BAZ = 2;
}

```
```
message Nested {
    enum Foo {
        FOO_TEST = 0;
        FOO_TEST2 = 1;
    }
    enum Nested_Foo {
        NESTED_FOO_TEST=0;
        NESTED_FOO_TEST2=1;
    }
    enum Nested_Bar {
        NESTED_BAR_TEST=0;
        NESTED_BAR_TEST2=1;
    }
}
enum Bar {
    BAR_UNSPECIFIED = 0;
    BAZ = 1;
    QUX = 2;
}
```

outputs as 
```

export enum Foo {
  UNSPECIFIED = 0,
  BAR = 1,
  BAZ = 2,
}

export enum Bar {
  UNSPECIFIED = 0,
  BAZ = 1,
  QUX = 2,
}

export interface Nested {
}

export enum Nested_Foo {
  FOO_TEST = 0,
  FOO_TEST2 = 1,
}

export enum Nested_NestedFoo {
  NESTED_FOO_TEST = 0,
  NESTED_FOO_TEST2 = 1,
}


export enum Nested_NestedBar {
  NESTED_BAR_TEST = 0,
  NESTED_BAR_TEST2 = 1,
}

```

I think that the behaviour is correct for the Nested_Foo enum, whereas it should output 
```
export enum Nested_NestedFoo {
   TEST = 0;
   TEST2 = 1;
}

```
Is this an issue or did you plan the feature like this?

**Comment by stephenh** [comment]
@Frabat ah no, I don't think the output you're seeing is intentional, probably just the first time someone is using `removeEnumPrefix` in nested enums.

If you'd like submit a PR that updates the behavior for nested enums, that'd be great!

**Comment by Frabat** [comment]
@stephenh I'll give it a try :)

**Comment by stephenh** [comment]
This was fixed by #903 , thanks @buzap !
